src/sentry/conf/server.py

```diff
@@ -1572,6 +1572,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
 SENTRY_METRICS_SAMPLE_RATE = 1.0
 SENTRY_METRICS_PREFIX = "sentry."
 SENTRY_METRICS_SKIP_INTERNAL_PREFIXES = []  # Order this by most frequent prefixes.
+SENTRY_METRICS_WARN_BAD_TAGS = IS_DEV
```

Comment:
My memory here is that `IS_DEV` is not actually set in tests :( I have vague memories of matt attempting to fix this and it breaking. Let's explicitly set this true in `pytest_configure`?